notebooks/mnist.ipynb

```diff
@@ -0,0 +1,113 @@
+      "text/plain": [
+       "<Figure size 1200x700 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(figsize=(6, 3.5))\n",
+    "ax.hist(zs, bins=10, density=True)\n",
```

Comment:
Should plot `zs[:,0] / zs[:,1]` since the dof will be different for each sample. Or you could use pvalue....